jenkins/release.jenkinsFile

```diff
@@ -0,0 +1,35 @@
+lib = library(identifier: 'jenkins@4.2.2', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+standardReleasePipelineWithGenericTrigger(
+    tokenIdCredential: 'jenkins-sql-odbc-generic-webhook-token',
+    causeString: 'A tag was cut on opensearch-project/sql-odbc repository causing this workflow to run',
+    downloadReleaseAsset: true,
+    publishRelease: true) {
+                        publishToArtifactsProdBucket(
+                            assumedRoleName: 'sql-odbc-upload-role',
+                            source: 'opensearch-sql-odbc-driver-32-bit-1.5.0.0-Windows.msi',
```

Comment:
Do this source names match the actual artifact? This was a suggestion based on actual artifact names which had spaces in them before.

Reply:
Looks like not https://github.com/opensearch-project/sql-odbc/pull/52#discussion_r1235911720

Reply:
Oh I see, thanks. Does S3 bucket support spaces in filenames?

Reply:
I think it does but signing and underlying linux activities might cause issues. Is it possible to fix those names? Just to be safe for other activities as well.

Reply:
Ack. Going to rename files.

Reply:
Fixed in 92bdad8.